docs/developers/pf-configuration.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Price Feed Configuration
+sidebar_label: Price Feed Configuration
+---
+
+Price feeds allow the creation of bots that can programmatically calculate prices off-chain and use this information to propose on optimistic price requests, dispute price proposals, monitor and liquidate sponsor positions or dispute liquidations. In the event of disputes being brought up to DVM for resolution price feeds can also assist UMA voters in their initial evaluation, though voters should always use human judgement to interpret obtained results.
+
+Availability of price feed is an absolute must for liquidable contracts and they should also generally be used by any other contracts expecting quick settlement if price fetching is relatively straightforward. The only exceptions where implementing a price feed might not be practical are KPI options or tracking more complex index data requiring involved manual calculation.
+
+## Selecting a Price Feed
+
+Product developers should document their price feed implementation choice and configuration when writing UMIP for new price identifier or passing price feed configuration as ancillary data for a generic price identifier. [UMA protocol repo](https://github.com/UMAprotocol/protocol/tree/master/packages/financial-templates-lib/src/price-feed) contains number of price feed implementations and this documentation guides on selecting and configuring those that could be more generally applicable for most applications.
```

Comment:
Is the plan to expand the instructions on setting ancillary data for the `TOKEN_PRICE` identifier once it is voted in? Or do you think that belongs in a different section or is unnecessary?

Reply:
Will add the link to UMIP once its voted in.
Will merge now